rustls/src/tls13/mod.rs

```diff
@@ -142,30 +143,57 @@ fn make_tls13_aad(len: usize) -> ring::aead::Aad<[u8; TLS13_AAD_SIZE]> {
 // https://datatracker.ietf.org/doc/html/rfc8446#section-5.2
 const TLS13_AAD_SIZE: usize = 1 + 2 + 2;
 
-struct AeadAlgorithm(&'static ring::aead::Algorithm);
+struct AeadAlgorithm {
```

Comment:
(More ordering...) The `struct` definition here should go belong the `static`s that use it.